specification/storage/resource-manager/readme.md

```diff
@@ -29,6 +29,33 @@ openapi-type: arm
 tag: package-2019-06
 ```
 
+### Tag: package-2021-01
```

Comment:
If you want to make the new tag as default one, please change default tag to `package-2021-01`

Reply:
Done.

Reply:
Yes, the default tag should change the the latest  GA API version